balancer-js/src/modules/data/index.ts

```diff
@@ -33,6 +37,8 @@ import initialCoingeckoList from '@/modules/data/token-prices/initial-list.json'
 export class Data implements BalancerDataRepositories {
   pools;
   yesterdaysPools;
+  poolShares;
+  gaugeShares;
```

Comment:
we need to add these to the `BalancerDataRepositories` type as well:
```
poolShares: Findable<PoolShare, PoolShareAttribute>;
gaugeShares: Findable<GaugeShare, GaugeShareAttribute>;
```
and update the data test/factories/data.ts

Reply:
it should be ok